### ISSUE TITLE
Windows menu shortcuts not supported with micromamba

### ISSUE THREAD

**Issue by yokoshin** [conda-forge/miniforge#158]
Hi, I have a question.

In miniforge  4.10.1-1, 
https://github.com/conda-forge/miniforge/releases/tag/4.10.1-1
The size of "Miniforge3-Windows-x86_64.exe" is 60.0 MB

In miniforge  4.10.1-3, 
https://github.com/conda-forge/miniforge/releases/tag/4.10.1-3
The size of "Miniforge3-Windows-x86_64.exe" is 53.6 MB

The package became much smaller and its install became faster.
Are there any big changes between the two?

**Comment by isuruf** [comment]
The conda bootstrapper was changed from `conda-standalone` to `micromamba`. This change only affects the extracting of the tarballs during the installation of miniforge and the created base environment should be the same.

**Comment by yokoshin** [comment]
Thank you for answering.
I encountered an error when creating menu in windows.
So I wondered why such a noticable bug was inclucded.

**Comment by isuruf** [comment]
@wolfv, does micromamba support menuinst shortcuts? If not, we might have to revert the micromamba usage on windows.

**Comment by wolfv** [comment]
I don't think we currently support the `menuinst` action. We'll have to look into it – I'll file an issue on the mamba-repo.

**Comment by wolfv** [comment]
micromamba issue opened here: https://github.com/mamba-org/mamba/issues/961
